### PR TITLE
Have APNSCertificate change events use a JMS topic instead of CDI event

### PR DESCRIPTION
**Pull request by danielpassos**
Related issue: [AEROGEAR-9068](https://issues.jboss.org/browse/AEROGEAR-9068)

## How to test?

1. Build the project ignoring the tests
   ```shell
   mvn clean install -DskipTests
   ```
1. Add a new new instance of UniedPush Server in `docker-compose/docker-compose-v2.1.yaml` 
   ```yaml
     unifiedpushserver_2:
    image: aerogear/unifiedpush-wildfly-plain:2.2.2-SNAPSHOT
    volumes:
       - ./helper:/ups-helper:z
    entrypoint: "/ups-helper/exportKeycloakHost.sh"
    depends_on:
       unifiedpushDB:
         condition: service_healthy
    environment:
        POSTGRES_SERVICE_HOST: ${POSTGRES_SERVICE_HOST}
        POSTGRES_USER: ${POSTGRES_USER}
        POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
        POSTGRES_SERVICE_PORT: ${POSTGRES_SERVICE_PORT}
        POSTGRES_DATABASE: ${POSTGRES_DATABASE}
        KEYCLOAK_SERVICE_HOST: ${KEYCLOAK_SERVICE_HOST}
        KEYCLOAK_SERVICE_PORT: ${KEYCLOAK_SERVICE_PORT}
    links:
      - unifiedpushDB:unifiedpush
      - artemis:artemis
      - keycloakServer:keycloak
    ports:
      - 9998:8080      
   ```
1. Run the docker compose. It will run 2 instance of the UPS point to the same postgres and ActiveMQ simulating 2 Openshift pods.
   ```shell
   cd docker-compose && docker-compose -f ./docker-compose-v2.1.yaml up
   ```
1. Create a new project and a iOS variant.
1. Edit cert in the iOS variant (you can change it for the same with the previous one) 
1. See the magics happening between the 2 UPS instance
1. You can also check the ActiveMQ in `http://localhost:8161`. (artemis/simetraehcapa)

### CODE REVIEW

File: docker-compose/docker-compose-v2.1.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: '2.1'
 
 services:
   unifiedpushserver:
-    image: docker.io/aerogear/unifiedpush-wildfly:2.1.0
+    image: aerogear/unifiedpush-wildfly-plain:2.2.2-SNAPSHOT
```
Comment (grdryn) [inline docker-compose/docker-compose-v2.1.yaml]: Not part of this PR, but I think it would be good to start publishing this to https://quay.io/organization/aerogear instead

Comment (danielpassos) [reply docker-compose/docker-compose-v2.1.yaml]: Good catch, we can change it before merge, anyway it's using temp SNAPSHOT and will be replaced by the released one!

File: model/jpa/pom.xml [hunk 2]
```diff
@@ -82,7 +82,12 @@
                     <artifactId>hibernate-entitymanager</artifactId>
                     <scope>test</scope>
                 </dependency>
-        
+        <dependency>
```
Comment (grdryn) [inline model/jpa/pom.xml]: The indentation seems a little off here.

File: push-sender/src/main/java/org/jboss/aerogear/unifiedpush/message/cache/SimpleApnsClientCache.java [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
+ * <p>
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- *
- * 	http://www.apache.org/licenses/LICENSE-2.0
- *
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
```
Comment (grdryn) [inline push-sender/src/main/java/org/jboss/aerogear/unifiedpush/message/cache/SimpleApnsClientCache.java]: I guess these are html paragraph tags (and probably generated by something), are they? Does the fact that there are 3 of them mean that one is mismatched? :thinking: Or are they all mismatched (should they have closing `</p>` -- I can't really remember html, but I think it's usually obscenely forgiving with that kind of thing)?

**Comment by secondsun** [comment]
@danielpassos I included this pr in https://github.com/aerogear/aerogear-unifiedpush-server/pull/1019
